static/js/modules/wikipediaAPI/util.js

```diff
@@ -71,4 +71,8 @@ async function getArticleSummary(page) {
     return body
 }
 
-export { getArticle, getArticleTitle, getArticleSummary, articleCheck };
\ No newline at end of file
+function round(value, decimals) {
```

Comment:
This shouldn't be in `wikipediaAPI/util.js`, these are meant to work with the wikipedia API, not our own. Also wow this is really the best way to round?

Reply:
I don't know what I was doing here